packages/@aws-cdk/aws-events-targets/test/logs/log-group.test.ts

```diff
@@ -54,9 +56,9 @@ test('use log group as an event rule target', () => {
   });
 });
 
-test('use log group as an event rule target with rule target input', () => {
+testLegacyBehavior('use log group as an event rule target with rule target input', cdk.App, (app) => {
```

Comment:
I don't think we want to use this. These tests won't be run at all in v2. Same with all the instances below.